.circleci/config.yml

```diff
@@ -68,6 +68,16 @@ jobs:
       - run:
           name: Updating configuration and database
           command: ./scripts/start.sh
+  cron:
+    docker:
+      - image: govau/cf-cli
+    working_directory: /app
+    steps:
+      - attach_workspace:
```

Comment:
btw I'm not actually sure if this `attach_workspace` will work here - usually the workspace is saved in the 'test' job,  and Im not sure if this will work in this context.